src/Microsoft.AspNetCore.Http.Connections.Client/TransportFailedException.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Http.Connections.Client
+{
+    public class TransportFailedException : Exception
+    {
+        public string TransportType { get; }
+
+        public TransportFailedException(string transportType, string message, Exception innerException = null)
+            : base($"{transportType} transport failed. {message}", innerException)
```

Comment:
```suggestion
            : base($"{transportType} failed: {message}", innerException)
```

Make it look like the JS error, just so it's somewhat consistent.

Reply:
(I'm fine if you'd rather change the JS one to look like this, just make them consistent :).